android/src/main/java/com/dooboolab/fluttersound/FlutterSoundPlugin.java

```diff
@@ -108,7 +110,7 @@ public boolean onRequestPermissionsResult(int requestCode, String[] permissions,
   }
 
   @Override
-  public void startRecorder(String path, final Result result) {
+  public void startRecorder(int numChannels, int sampleRate, String path, final Result result) {
```

Comment:
I hope you to provide this parameter optionally.
Like
startRecorder(
  String path, final Result result,
  numChannelsAndroid: 1,
  sampleRateAndroid: 44100,
) {
...
}